testonly/hammer/hammer.go

```diff
@@ -338,6 +416,9 @@ func newWriteWorker(s *hammerState) *writeWorker {
 	}
 }
 
+// loop continuously performs mutation operations on the map until the done signal
+// is received via th channel, an error is encountered, or the maximum number of
```

Comment:
simply: `... until the done channel is closed, an error ...`
Same for `readWorker`.

Reply:
Done.